.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "docs\\env\\Scripts\\python.exe"
+}
```

Comment:
I don't see the particular use-case of this here

Reply:
no this was for the installation part I need to set the virtual environment for that it needed this in windows 
![image](https://user-images.githubusercontent.com/72685035/134820003-7a2b11e6-5383-47ff-b0e8-1f75f9296495.png)